README-Gradle.md

```diff
@@ -96,6 +96,15 @@ plugin](https://docs.gradle.org/current/userguide/eclipse_plugin.html).
 A few WALA sub-projects already use this:  look for `eclipse.project`
 in `*/build.gradle.kt` for examples.
 
+### Visual Studio
+
+<img width="393" alt="Screenshot 2024-11-15 at 4 22 59 PM" src="https://github.com/user-attachments/assets/3824add4-f212-487e-9dd2-371c4a3770d8"> 
+<img width="407" alt="Screenshot 2024-11-15 at 4 22 47 PM" src="https://github.com/user-attachments/assets/e2961958-7d67-4694-94a4-be3aaf8f0d6a"> <br />
+Add the following extensions for *gradle* and *java* on VSCode at the market place. If you have already `built` the repo, clone the repo again. 
```

Comment:
Remove backticks around “built”: this word does not represent the literal text input one would type or the literal text output of any computational process. I would agree with putting backticks around `build`, since that is a Gradle task name, but “built” is not.